src/main/resources/assets/immersiverailroading/rolling_stock/default/base.caml

```diff
@@ -48,6 +48,7 @@ properties =                # Required:
 #       -1.0 = immersiverailroading:some/path/animation_-1.0.anim # Reverse
 #       0.0 = immersiverailroading:some/path/animation_0.0.anim # Neutral
 #       1.0 = immersiverailroading:some/path/animation_1.0.anim # Forward
+    left_first = true            # Required if valve_gear is defined: Should this valve gear be at left-first crank pin position? Set false to make it right-first
```

Comment:
What are your thoughts on changing this to a suffix on the valve_gear property?

valve_gear = WALSCHAERTS_INVERT

That's pretty easy to detect in ValveGearConfig.  This flag is also not applicable for the animatrix valve gear type, which this change would make clear.

Reply:
That's a fantastic proposal. I'll try it